test/sharness/t0030-mount.sh

```diff
@@ -53,6 +55,12 @@ test_expect_success FUSE "'ipfs mount' output looks good" '
   test_cmp expected actual
 '
 
+test_expect_success FUSE "can resolve ipns names" '
```

Comment:
Continuing from https://github.com/ipfs/go-ipfs/pull/7331#issuecomment-631180756 so we can do this in a thread-like fashion.

Running `./t0030-mount.sh -v` passes on master, but with this branch fails with:
`cat: ipns/welcome.example.com/ping: No such file or directory`

Reply:
Did you make sure to rebuild? Otherwise, I have no idea. Does `ipns/local` exist when running the test?

Reply:
I think I had a rebuilding issue... so dumb. Sorry for the confusion